### PR TITLE
Update the multicell migration page with eng clarifications

### PR DESCRIPTION
**Pull request by eiof**
## What are you changing in this pull request and why?

We recently updated the migration checklist in #6002. Engineering has reviewed the changes, and recommends clarifying some points in attempt to be more accurate and reassuring to our customers.

- An account is migrated on or after its scheduled date
- Accounts that have already been assigned a static subdomain...
  - will be able to use existing user invitations
  - will not lose unsaved IDE changes
- Cold-storage logs aren't lost, just not available through the user interface

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/migration.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
...
-- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes will be lost during migration.
-- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they will be lost during migration. If you lose critical logs older than 90 days during the migration, you will have to work with the dbt Labs Customer Support team to recover.
+- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations may be voided during the migration. You can resend user invitations after the migration is complete.
+- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes may be lost during migration.
+- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they may be unavailable from dbt Cloud after the migration is complete. Logs older than 90 days while within the data retention period are not deleted, but you will have to work with the dbt Labs Customer Support team to recover.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]: "Unavailable" reads like they could be made available again in the product. Can we use language that firmly states they are gone? Or does contacting support put the logs back in their UI? If not, let's be direct. We don't have to use lost, maybe deleted, purged, etc.

File: website/docs/docs/cloud/migration.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
 
 We're scheduling migrations by account. When we're ready to migrate your account, you will receive a banner or email communication with your migration date. If you have not received this communication, then you don't need to take action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
 
-Your account will be automatically migrated on its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
+Your account will be automatically migrated on or after its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md:14]: If it's "on or after," can we provide a time frame (within a hour/day/week of the date)?

File: website/docs/docs/cloud/migration.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
...
 
 ## Recommended actions
 
 We highly recommended you take these actions:
 
-- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations will be voided during the migration and must be resent after it is complete.
-- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes will be lost during migration.
-- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they will be lost during migration. If you lose critical logs older than 90 days during the migration, you will have to work with the dbt Labs Customer Support team to recover.
+- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations may be voided during the migration. You can resend user invitations after the migration is complete.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]:
```suggestion
- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations might be voided during the migration. You can resend user invitations after the migration is complete.
```

File: website/docs/docs/cloud/migration.md [hunk 4]
```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
...
+- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes may be lost during migration.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/migration.md]:
```suggestion
- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes might be lost during migration.
```

File: website/docs/docs/cloud/migration.md (hunk 2 above)
Comment (ryancharris) [reply website/docs/docs/cloud/migration.md:14]: @matthewshaver Right now, no. Accounts are eligible for migration when they're scheduled; however, in the window between scheduling and executing the migration, an account can become ineligible for a number of reasons. How long it takes for them to become eligible again depends on what/how much work is required to support the specific use-case.

File: website/docs/docs/cloud/migration.md [hunk 5]
```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
...
-- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes will be lost during migration.
-- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they will be lost during migration. If you lose critical logs older than 90 days during the migration, you will have to work with the dbt Labs Customer Support team to recover.
+- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations might be voided during the migration. You can resend user invitations after the migration is complete.
+- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes might be lost during migration.
+- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they may be unavailable from dbt Cloud after the migration is complete. Logs older than 90 days while within the data retention period are not deleted, but you will have to work with the dbt Labs Customer Support team to recover.
```
Comment (ryancharris) [inline website/docs/docs/cloud/migration.md]:
```suggestion
- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they may be unavailable from dbt Cloud after the migration is complete. Logs older than 90 days (while within the data retention period) are not deleted, but you will have to work with the dbt Labs Customer Support team to recover.
```

**Review by ryancharris — APPROVED** [review]

File: website/docs/docs/cloud/migration.md (hunk 1 above)
Comment (eiof) [reply website/docs/docs/cloud/migration.md]: I think the best description is "will be unavailable from dbt Cloud". I will update this.

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**